src/has.ts

```diff
@@ -169,10 +169,10 @@ add('xhr2-blob', function () {
 	return request.responseType === 'blob';
 });
 // Native Array methods
-add('array-from', 'from' in global.Array);
-add('array-of', 'of' in global.Array);
-add('array-fill', 'fill' in global.Array.prototype);
-add('array-findIndex', 'findIndex' in global.Array.prototype);
-add('array-find', 'find' in global.Array.prototype);
-add('array-copy-within', 'copyWithin' in global.Array.prototype);
-add('array-includes', 'includes' in global.Array.prototype);
+add('es6-array-from', 'from' in global.Array);
+add('es6-array-of', 'of' in global.Array);
+add('es6-array-fill', 'fill' in global.Array.prototype);
+add('es6-array-findIndex', 'findIndex' in global.Array.prototype);
+add('es6-array-find', 'find' in global.Array.prototype);
+add('es6-array-copyWithin', 'copyWithin' in global.Array.prototype);
+add('es7-array-includes', 'includes' in global.Array.prototype);
```

Comment:
It is officially in ES2016/7, only that and the exponentiation operator. :grinning: